llvm/test/tools/llvm-objdump/X86/elf-pgoanalysismap.yaml

```diff
@@ -104,12 +106,22 @@ Symbols:
 # ENTRYCOUNT-BLOCKFREQ: <BB2> (Frequency: 18):
 # ENTRYCOUNT-BLOCKFREQ: <BB5> (Frequency: 1000):
 
+# ENTRYCOUNT-BLOCKFREQ-PRETTY: <foo>:
+# ENTRYCOUNT-BLOCKFREQ-PRETTY: <BB3> (Entry count: 1000, Frequency: 1.0):
```

Comment:
Are these on consecutive lines in the output?

Reply:
These first two are on consecutive lines but the others are not.